package.json

```diff
@@ -58,7 +58,7 @@
     "lint:path": "eslint --ignore-path .eslintignore --ext .js,.ts,.tsx",
     "prettier": "prettier",
     "jest": "jest",
-    "test": "turbo run test --concurrency=50% --no-daemon --no-cache --force --filter='./packages/*' --filter='./packages/core/*' --filter='./packages/cli/*' --filter='./packages/modules/*' --filter='./packages/modules/providers/*'",
+    "test": "turbo run test --concurrency=50% --no-daemon --no-cache --force",
```

Comment:
Q: I remember we added those filters to be sure it only runs the unit tests from packages but that the filter did not allow for deep regexp, are we sure only the unit tests are ran from the packages directory here?

Reply:
the .sh script will run all packages in the workspace. Which ones need to be filtered out?

Reply:
oh ok perfect